### DISCUSSION TITLE
speedup table

### DISCUSSION THREAD

**Discussion by shurshilov** [icflorescu/mantine-datatable#664] [General]
![image](https://github.com/user-attachments/assets/12139ac9-4dca-411a-a065-043c82768f18)
already on 1000 records, the selected record has a delay of 2-3 seconds, and in general, braking is noticeable on 5000, it is already very difficult to use the table on 10K is almost impossible.
At the same time, antd has a virtual option and it works instantly on 10K. Maybe we can somehow improve the speed of the work?

**Comment by petrvecera** [comment]
Probably the same issue as here https://github.com/icflorescu/mantine-datatable/discussions/643
